index.js

```diff
@@ -22,33 +23,37 @@ module.exports = (robot, options) => {
 
   // https://developer.github.com/v3/activity/events/types/#installationrepositoriesevent
   robot.on('installation_repositories.added', async event => {
-    const installation = event.payload.installation
-
-    // FIXME: get added repositories from webhook
-    eachRepository(installation, repository => {
-      if (!intervals[repository.id]) {
-        schedule(installation, repository)
-      }
-    })
+    return setupInstallation(event.payload.installation)
   })
 
   setup()
 
   function setup () {
-    eachInstallation(installation => {
-      limiter.schedule(eachRepository, installation, repository => {
-        schedule(installation, repository)
-      })
+    return eachInstallation(setupInstallation)
+  }
+
+  function setupInstallation (installation) {
+    if (ignoredAccounts.includes(installation.account.login)) {
```

Comment:
I assume that `installation.account.login` are guaranteed to always be lowercase, but I never tested it myself